doc/JA/dev_guide/commit_message.md

```diff
@@ -29,13 +29,31 @@ commit ::= type ('(' scope ')')? '!'? ':' description ('(' '#' issue ')')? body?
 
 各部分の意味は以下の通りです。
 
-* `type`はcommitの型を表します。小文字で書いてください(自動commitは大文字で始まるので、これによって手動コミットかどうかを区別します)。`feat`は新しい機能、`fix`はバグの修正やissueの解決、`docs`はドキュメントの変更、`style`はコードスタイルの変更、`refactor`はリファクタリング、`perf`はパフォーマンスの改善、`test`はテストの追加や変更、`build`はビルド関連・バージョン・依存関係の変更、`ci`はCI関連の変更、`chore`は内部的・軽微な変更、`revert`はrevertです。複数該当する場合は、より具体的なtypeを選んでください。優先度の低いtypeは`fix`, `refactor`, `style`, `chore`になります。例えば、ドキュメント(docs)の修正(fix)は`docs`、テスト(test)のリファクタリング(refactor)は`test`になります。
+* `type`はcommitの型を表します。小文字で書いてください(自動commitは大文字で始まるので、これによって手動コミットかどうかを区別します)
+
+| type       | 説明                                   |
+| ---------- | -------------------------------------- |
+| `feat`     | 新しい機能                             |
+| `fix`      | バグの修正やissueの解決                |
+| `docs`     | ドキュメントの変更                     |
+| `style`    | コードスタイルの変更                   |
+| `refactor` | リファクタリング                       |
+| `perf`     | パフォーマンスの改善                   |
+| `test`     | テストの追加や変更                     |
+| `build`    | ビルド関連・バージョン・依存関係の変更 |
+| `ci`       | CI関連の変更                           |
+| `chore`    | 内部的・軽微な変更                     |
+| `revert`   | revert                                 |
+
```

Comment:
The auto-formatting doesn't seem to work well for full-width chars

Reply:
Since full-width and half-width characters are mixed, the widths cannot be aligned. But no problem with Markdown rendering.